package.json

```diff
@@ -13,15 +13,17 @@
     "coverage": "IASQL_ENV=test jest -i --coverage --forceExit",
     "coverage:local": "IASQL_ENV=test export $(cat .env | xargs) && yarn coverage",
     "serve": "node --max_old_space_size=8192 dist/index.js",
-    "serve:debug": "node --inspect=0.0.0.0:9229 --max_old_space_size=8192 dist/index.js",
+    "serve:debug": "yarn build && node --max_old_space_size=8192 dist/index.js",
     "start": "yarn serve",
-    "start:debug": "nodemon -e ts -w ./src -x yarn serve:debug",
+    "start:debug": "nodemon --inspect=0.0.0.0:9229 -e ts -w ./src -x yarn serve:debug",
     "gen-sql": "IASQL_ENV=ci ts-node ./node_modules/.bin/typeorm migration:generate -n",
     "migrate": "IASQL_ENV=ci ts-node ./node_modules/.bin/typeorm migration:run",
     "gen-module": "IASQL_ENV=ci ./.module_migration.sh",
     "docker-build": "docker build -t iasql:latest .",
+    "docker-build:dev": "docker build -f ./Dockerfile.dev -t iasql:latest-dev .",
     "docker-run": "docker run -p 5432:5432 --name iasql iasql:latest",
     "docker-run:detach": "docker run -d -p 5432:5432 --name iasql iasql:latest",
+    "docker-run:dev": "docker run -p 5432:5432 -p 9229:9229 --name iasql-dev -v $PWD/src:/engine/src -v /engine/node_modules iasql:latest-dev",
```

Comment:
`-v /engine/node_modules` going to make issues if the host and the docker container are from two different architectures.

Reply:
I think you're not overwriting the local `node_module`. Just signaling the persistence, right?

In that case it's ok